config.py

```diff
@@ -144,6 +163,9 @@ class Configuration(object):
             "key": BASE_URL_KEY,
             "label": _("Base url of the application"),
         },
+        { "key": LOG_LEVEL, "label": _("Log Level"), "type": "select", "options": LOG_LEVEL_UI },
+        { "key": LOG_APP_NAME, "label": _("Log App") },
```

Comment:
This could use a better label or a description, I wouldn't know what it is.

Reply:
Ah, I see there is one in LogConfiguration. I'm not sure why this setting is in two places though.

Reply:
This may be from some confusion on my part. But I moved these values to LogConfiguration which is where it gets picked up from.